zsh/beamery/plugins/clean_git_stash.zsh

```diff
@@ -9,7 +9,7 @@ source "${HOME}/.oh-my-zsh/plugins/beamery/pluginsInterface.zsh"
 clean_git_stash() {
 
     printf "${RED}Please note that this will clear any stashes you have saved${NC}\n"
-    read -p "Are you sure you want to proceed ? [Y/N] " -n 1;
+    read -q "REPLY?Are you sure you want to proceed ? [Y/N] " 1;
```

Comment:
no need for the extra `1` at the end .. same thing for all the other files .. other than it LGTM